src/test/java/edu/usf/cutr/test/ResultAnalyzerTest.java

```diff
@@ -0,0 +1,384 @@
+package edu.usf.cutr.test;
```

Comment:
Package (and directory structure) for this file should be `edu.usf.cutr.transitfeedqualitycalculator.test`.